middle_end/flambda2/to_cmm/to_cmm_expr.ml

```diff
@@ -129,9 +141,16 @@ let translate_apply0 env res apply =
         return_arity |> Flambda_arity.With_subkinds.to_arity
         |> C.machtype_of_return_arity
       in
+      let _args_ty =
+        List.map
+          (fun k ->
+            To_cmm_utils.machtype_of_kind (Flambda_kind.With_subkind.kind k))
+          (Flambda_arity.With_subkinds.to_list param_arity)
+      in
```

Comment:
I suppose these are to be used later when indirect call need the layouts of the args to generate the call correct `caml_apply_*` ? If so, you could add a comment/CR to make it clear (that also applies to the occurrence of `_args_ty` above).